### DISCUSSION TITLE
Sum a row in budibase

### DISCUSSION THREAD

**Discussion by philiplorenz-code** [Budibase/budibase#11871] [Help]
Hello together,

I want to ask whether it is possible to create a new column, that displays the sum of specific items in a row. my table has following schema:

date - c1 - c2 - c3
2023/05/12 - 4 - 5 - 3
2023/05/13 - 2 - 1 - 6

is it possible to insert another column that always shows the sum of c1 and c3? like this:
date - c1 - c2 - c3 - sum
2023/05/12 - 4 - 5 - 3 - 7
2023/05/13 - 2 - 1 - 6 - 8

I tried to use Formula column but was not able to achieve this. Thanks in advance :)

**Comment by MikeYates** [comment]
One way I do this is by using SuperTable, since I use an external database, but internal will work too.

Install SuperTable plugin, and SuperTable cell. Insert the columns and align according to your old table. Insert a data provider in the column you want to show the value, filter it according to the row id, insert repeater inside the data provider, insert a SuperTable cell inside the repeater, click on the cell value binding, use the Add helper `{{ Add Column1 Column2 }}`

That should work.
